cypress/e2e/supervision/orders/bonds/edit-bond.cy.js

```diff
@@ -0,0 +1,33 @@
+beforeEach(() => {
+  cy.loginAs("Case Manager");
+  cy.createClient()
+    .withOrder()
+    .withBond();
+});
+
+describe(
+  "Edit bond",
+  () => {
+    Cypress._.times(50, () => {
```

Comment:
probably want to take the 50 times run out before the merge

Reply:
oops nice catch thank you